opentelemetry-api/opentelemetry/backend.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+The OpenTelemetry backend module defines and implements the API to access an
+implementation (backend) of the OpenTelemetry API.
```

Comment:
Do we need this to be an API too? I wouldn't expect users to want to replace the loader.

Reply:
I'm not sure I properly understand what you mean here. But based on what I *think* you mean, I say: The term "API" here isn't meant to indicate that the backend module can be replaced by the user. That's why the docstring says "defines **and implements**".  But maybe "API" in this context really is confusing and anyway, "The OpenTelemetry backend module **provides access to** an implementation (backend) of the OpenTelemetry API." is a simpler sentence.